node/test/util/cherry_pick.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2020, Two Sigma Open Source
```

Comment:
2021?

Reply:
I wrote it in 2020 but forgot to check it in then.

Reply:
oops, actually it should be 2021 -- publication date.  will fix.